### DISCUSSION TITLE
S1 is not updating when attempting to install the universal firmware

### DISCUSSION THREAD

**Discussion by Geo-Warrior** [mriscoc/Ender3V2S1#116] [Q&A] [answered]
Hello. When I insert a properly formatted and configured SD card into my S1 there is no 10 second delay and the printer boots right up. I've tried a couple different SD cards with the same outcome. Is this an indication that I have the incompatible board by chance. God I hope not.

**Comment by mriscoc** [comment, marked as answer]
Hi, go to Control / Info and post your current firmware version

**Reply by Geo-Warrior** [reply]
It says 3.0.3. I think I saw reference to that number earlier.

**Reply by mriscoc** [reply]
Sorry, you have a board with a STM32F4xx. The current firmware version is incompatible with this SoC, only boards with STM32F1xx are supported.

**Reply by Geo-Warrior** [reply]
ugh...it's always something it seems. thank you for the quick response. Much appreciation for all you do for the community.

**Reply by Geo-Warrior** [reply]
Do you have any idea of what the likelihood might be of getting the STM32F1xx board if I were to request a return/replacement through Amazon? Also, I just purchased a BTT SKR mini E3 V3. Would that be compatible with the S1 as a replacement board? Using a different firmware I would assume? Thanks again.

**Reply by mriscoc** [reply]
About of STM32F1xx board, I don't know.
About of BTT SKR mini E3 V3, I don't have the configuration files for that board.
